cdk/makerspace.py

```diff
@@ -46,6 +49,13 @@ def database_stack(self):
 
         self.database = Database(self.app, self.stage, env=self.env)
 
+        self.plan = aws_backup.BackupPlan.daily_monthly1_year_retention(self, 'BackupPlan')
```

Comment:
I must have missed the PITR when I was looking at the DynamoDB backup info, but it definitely seems like PITR would be a more useful solution for backing up the DB so I'll definitely look into how to implement that!

Reply:
Sweet! In addition to this- I would highly suggest setting these values as well:

1. Set the CloudFormation resource deletion policy to `RETAIN`
    - CloudFormation is AWS's "Infrastructure as Code" service. You write a json or yaml template which are effectively instructions for CloudFormation to create/update/delete resources. CDK adds a layer of abstraction on top of raw CloudFormation by enabling you to write code that generates your CloudFormation template. What we want to do here is tell CDK to set an instruction in our CloudFormation template that says: "When I remove this resource from my cdk/CloudFormation template dont attempt to delete it. Just forget it exists and move on"
    - https://bobbyhadz.com/blog/set-deletion-policy-aws-cdk - `cdk.RemovalPolicy.RETAIN`

2. Set termination protection on our CloudFormation stacks
   - When you delete a cloudformation stack, cloudformation attempts to delete all of the resources that it manages. CloudFormation also offers the ability to protect a stack from being deleted. That ability is called "Termination Protection." If termination protection is set to on and you attempt to delete your stack, the deletion will fail. In order to delete the stack you would be required to first set the attribute to false and then retry the delete. This is known as "delete with friction."
   - https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_core.Stack.html - `terminationProtection = true`

Each of the above should be a single line change but they will each help to protect against the accidental deletion of our resources which would cause customer impact now that this app is live.

Reply:
No need to do all at once by the way- just want to give you a sense of the protections that *we* put in place as owners of AWS services.